app/src/main/res/values/styles.xml

```diff
@@ -17,15 +18,40 @@
         <item name="android:textSize">48sp</item>
     </style>
 
+    <style name="AppTheme.Title.TextAppearance" parent="android:TextAppearance">
+        <item name="android:textSize">30sp</item>
+        <item name="android:textColor">@color/textColorLight</item>
+        <item name="android:textStyle">bold</item>
```

Comment:
Хорошим тоном является именование ресурсов семантически. То есть использовать не сырые цвета (белый/серы итд) а выбирать названия исходя из семантики - то есть где они используются по смыслу.
Например вместо color_white = color_background_main, вместо color_grey = color_divider, color_blue= color_actor_card_background. Это так, FYI. Профит от этого например при добавлении темной темы - названия цветов должны быть универсальны, как я описал выше, а на уровне ресурсов темы они уже меняются. Хорошая статья на эту тему https://habr.com/ru/company/redmadrobot/blog/523916/
Это не обязательно сейчас так делать - но просто опять таки одна из идей для улучшения

Вот даже тут описаны частые ошибки: https://developer.android.com/guide/topics/ui/look-and-feel/darktheme#widgets
Common pitfalls to look out for:

Assuming that the background color is always light
Hardcoding text colors
Setting a hardcoded background color, while using the default text color
Using a drawable icon which is a static color
In all of these cases, use appropriate theme attributes instead of hardcoded colors.